src/main.rs

```diff
@@ -495,10 +612,16 @@ fn main() -> anyhow::Result<()> {
                 .join()
                 .unwrap_or_else(|e| panic!("csi thread error: {:?}", e));
         }
-        RunAsRole::SchedulerExtender => {
+        (START_SCHEDULER_EXTENDER, Some(matches)) => {
+            let metadata = parse_metadata(matches, RunAsRole::SchedulerExtender)?;
+            let md = Arc::new(metadata);
+            let sep = get_scheduler_port(matches);
```

Comment:
change `sep` to `port` or `scheduler_port`. sep is not a good name.

Reply:
done